Fix test failure that occurs with recent versions of Flake8.

Taken from upstream:
https://github.com/Kozea/tinycss2/commit/6556604fb98c2153412384d6f0f705db2da1aa60

diff --git a/tinycss2/css-parsing-tests/make_color3_hsl.py b/tinycss2/css-parsing-tests/make_color3_hsl.py
index d1fd3a6..56fda0c 100644
--- a/tinycss2/css-parsing-tests/make_color3_hsl.py
+++ b/tinycss2/css-parsing-tests/make_color3_hsl.py
@@ -8,16 +8,17 @@ def trim(s):
 print('[')
 print(',\n'.join(
     '"hsl%s(%s, %s%%, %s%%%s)", [%s, %s, %s, %s]' % (
-        ('a' if a is not None else '', h,
-         trim(str(s / 10.)), trim(str(l / 10.)),
-         ', %s' % a if a is not None else '') +
+        ('a' if alpha is not None else '', hue,
+         trim(str(saturation / 10.)), trim(str(light / 10.)),
+         ', %s' % alpha if alpha is not None else '') +
         tuple(trim(str(round(v, 10)))
-              for v in colorsys.hls_to_rgb(h / 360., l / 1000., s / 1000.)) +
-        (a if a is not None else 1,)
+              for v in colorsys.hls_to_rgb(
+                hue / 360., light / 1000., saturation / 1000.)) +
+        (alpha if alpha is not None else 1,)
     )
-    for a in [None, 1, .2, 0]
-    for l in range(0, 1001, 125)
-    for s in range(0, 1001, 125)
-    for h in range(0, 360, 30)
+    for alpha in [None, 1, .2, 0]
+    for light in range(0, 1001, 125)
+    for saturation in range(0, 1001, 125)
+    for hue in range(0, 360, 30)
 ))
 print(']')
